src/app/models/ISell.ts

```diff
@@ -0,0 +1,9 @@
+import { ICake } from './Cake';
+
+export interface ISell {
+  customerName: string;
+  customerPhoneNumber: string;
+  customerEmail: string;
+  totalAmount: number;
+  cake: ICake;
```

Comment:
Be careful using the cake interface, since the cake interface has more members than need it at this point.

Reply:
Moving into a separate file